python_scripts/ensemble_bagging.py

```diff
@@ -257,11 +257,17 @@ def bootstrap_sample(data, target):
 # Let us compare the based model predictions with their average:
 
 # %%
-for tree_idx, tree in enumerate(bagged_trees.estimators_):
-    label = "Predictions of individual trees" if tree_idx == 0 else None
-    tree_predictions = tree.predict(data_test)
-    plt.plot(data_test, tree_predictions, linestyle="--", alpha=0.1,
-             color="tab:blue", label=label)
+import warnings
+
+with warnings.catch_warnings():
+    # call that will ignore the warning messages
```

Comment:
I think we can remove this line because the code will be self-explanatory.
Otherwise, we can specify really the type of warning that we want to ignore triggered by scikit-learn